### PR TITLE
Add broken_linkfinder code

### PR DESCRIPTION
**Pull request by harshitashankar**
# Pull Request Template

**script name** - broken_linkfinder.py

**Brief about script** Brief - A script that detects broken links in a website provided by the user (one can use requests, soupsieve or similar libs)

### Self Check(Tick After Making pull Request)

- [x] This issue was assigned to me.
- [x] One Change in one Pull Request
- [x] My file is in proper folder (Name of folder should be in lowercase with no space in between)
- [x] I am following clean code and Documentation and my code is well linted with flake8.
- [x] I have added README.md and requirements.txt with my script

### CODE REVIEW

**Review by Sacrezar — APPROVED** [review]
Nice work! Thank you 😃

File: broken_linkfinder/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+## Setup and activate virtual environment :
+For Unix based systems please execute the following command to create venv and install requirements.
+```
+make init
+source .venv/bin/activate
+```
+
+## Usage: 
+
```
Comment (pawangeek) [inline broken_linkfinder/README.md:11]: Add more about the purpose of the script and a brief description of how it gonna solve that

Comment (harshitashankar) [reply broken_linkfinder/README.md]: Updated.

**Review by pawangeek — APPROVED** [review]

**Comment by pawangeek** [comment]
@all-contributors add @harshitashankar for code

**Pull request merged**